membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -70,7 +70,7 @@ object AttributesFromZuora extends LoggingWithLogstashFields {
         val attributesAfterUpdate = for {
             attributes <- OptionT(zuoraAttributesWithAdfree)
           _ = dynamoAttributeUpdater(attributes).map { result =>
-            result.left.map { error: DynamoReadError => log.warn(s"Tried updating attributes for $identityId but then ${DynamoReadError.describe(error)}") }
+            result.left.map { error: DynamoReadError => log.error(s"Tried updating attributes for $identityId but then ${DynamoReadError.describe(error)}") }
           }.onFailure {
             case error => log.warn(s"Tried updating attributes for $identityId but then ${error.getMessage}", error)
```

Comment:
should be an error here really too

Reply:
do we need to be careful of PII though? we can error log (sentry) one vaguer thing and warn log the full details (kibana/cloudwatch)